src/utils/contribution.js

```diff
@@ -81,12 +81,16 @@ export default class Contribution {
   }
 
   /**
+   *
    * Applies changes to the specific component
+   * @static
    * @param  {} component original component
    * @param  {} changes object containing changes
    * @param  {} field field to check
    * @param  {} value value to apply to the field
    * @param  {} type used to specify a type of field (e.g. array) to perform different kind of operations
+   * @param {*} [transform=a => a]
+   * @returns
```

Comment:
@gianpaj can you confirm you wanted this @returns doc comment?

Reply:
that's what JSdoc added automatically. I've removed it